src/types/ros3d.d.ts

```diff
@@ -59,7 +57,6 @@ declare module 'ros3d' {
    */
   class Viewer {
```

Comment:
There's a warning saying that Viewer is never used.

Reply:
Everything in that file is just type definitions for typescript since the original library doesn't have them. I just copy pasted the documentation of every exported class. I'll leave it there because we might need it in the future.  I might just push them in https://github.com/DefinitelyTyped/DefinitelyTyped